downloads/index.pug

```diff
@@ -0,0 +1,53 @@
+---
+layout: default
+---
+
+// Inspired by: https://syncthing.net/downloads/
+
+article.container
+  h1 Downloads
+  // TODO: Link to GitHub Releases
+  // TODO: List all of the latest releases
```

Comment:
Link to GitHub releases should be enough, the list of all latest releases will be on the top anyway.

Reply:
Oops, those TODOs were from before I added them (I meant the latest windows/macos/linux/android releases).